### ISSUE TITLE
Here is an opportunity 

### ISSUE THREAD

**Issue by theatsign** [deanoemcke/thegreatsuspender#1042]
Please complete the following information when submitting a feature request or bug report.
 * Extension version:
 * Browser name & version:
 * Operating system & version:

And please also do a search for your request/bug before create a new one thanks!
Hello,
I put a bug in Viasats ear for you, currently testing the great suspender. 70% thru I am ready to recommend this app. Here is the survey response I sent them in response to the question, what they could do to improve their service,

Add an on screen data meter with settable limit reminders. Offer a tab suspender so tabs you are not viewing can not use data. I have a third party app that does this for me..... here is the writer's contact info... github.com/deanoemcke/thegreatsuspender. You could tie this to your browser and increase your traffic there!
To sum it up , anything you can do to help stay within my allowance!

I really feel good about this application, properly set up, it seems to dramatically reduce my data usage. Thank You !
